src/progressBar.js

```diff
@@ -18,7 +18,7 @@ class ProgressBarController {
     }
 
     // Don't do any of this while testing
-    if (process.env.NODE_ENV === "test") {
+    if (process.env.NODE_ENV === "lerna-test") {
```

Comment:
the progress bar does look a bit wonky in CI logs. wonder if there's a better flag here... maybe `env.CONTINUOUS_INTEGRATION` or something